input/pagecontent/reference.md

```diff
@@ -1544,25 +1651,103 @@ A defined rule set can be applied to an item by using an [`insert` rule](#rule-s
   * ^publisher = "Elbonian Medical Society"
   ```
 
-#### Defining Invariants
+#### Defining Value Sets
 
-Invariants are defined using the keywords `Invariant`, `Description`, `Expression`, `Severity`, and `XPath`. An invariant definition does not have any rules.
+A value set is a group of coded values, usually representing the acceptable values in a FHIR element whose data type is code, Coding, or CodeableConcept.
 
-Invariants are incorporated into a profile via `obeys` rules explained [above](#invariant-rules).
+Value sets are defined using the declarative keyword `ValueSet`, with optional keywords `Id`, `Title` and `Description`
 
-**Example:**
+Codes must be taken from one or more terminology systems (also called code systems or vocabularies). Codes cannot be defined inside a value set. If necessary, [you can define your own code system](#defining-code-systems).
 
-* Define an invariant found in US Core using FSH:
+The contents of a value set are defined by a set of rules. There are four types of rules to populate a value set:
+
+> **Note:** In value set rules, the word `include` is optional.
+
+| To include... | Syntax | Example |
+|-------|---------|----------|
+| A single code | `* include {Coding}` | `* include SCT#961000205106 "Wearing street clothes, no shoes"` <br/> or equivalently, <br/> `* SCT#961000205106 "Wearing street clothes, no shoes"`|
+| All codes from a value set | `* include codes from valueset {ValueSet name|id|url}` | `* include codes from valueset http://hl7.org/fhir/ValueSet/data-absent-reason`  <br/> or equivalently, <br/> `* codes from valueset http://hl7.org/fhir/ValueSet/data-absent-reason`|
```

Comment:
Just to be more clear, should it say "All codes from **another** value set"?

Reply:
Fixed in two places.